aws_cloudtrail_rules/aws_iam_anything_changed.yml

```diff
@@ -17,7 +17,6 @@ Reference: reference.link
 Tests:
   -
     Name: IAM Change
-    LogType: AWS.CloudTrail.Log
```

Comment:
Just noticed that this is still in the old format of `AWS.CloudTrail.Log`, both here and above under the still required top level `LogTypes`. Could you please replace all instances of `AWS.CloudTrail.Log` with `AWS.CloudTrail` under the `LogTypes` field in the top level?

Reply:
Seems like a minor-enough change to include it here to me.  Added